### ISSUE TITLE
v7.1.6.2 sometimes ignores form input

### ISSUE THREAD

**Issue by baz357** [gioxx/MarvellousSuspender#82]
I just noticed that a tab was suspended where I had written quite a bit of text. When I unsuspended the tab, Chrome did restore the text, but a while later it was suspended again and the text is gone now. :-/ (Should have saved it right away.)

Before the extension update I never had this happen, the icon would always change visually to 'disabled because of form input' when I started typing. (At least when I was paying attention.)

Latest Chrome and Windows versions here.

**Comment by gioxx** [comment]
Ciao @baz357 
Is the option "Never suspend tabs that contain unsaved form inputs" enabled? The behaviour is strange because we never changed the portion of code related to suspend a tab if you have filled a form.
Thanks.
Giovanni.

**Comment by baz357** [comment]
> Is the option "Never suspend tabs that contain unsaved form inputs" enabled?

Yes, it is. I just checked again to be sure.

> The behaviour is strange because we never changed the portion of code related to suspend a tab if you have filled a form.

Yeah, strange indeed... The icon has changed to 'disabled because of form input' for this tab, but when I try it with some forum in another tab it doesn't recognize the form input when I start typing.

**Comment by gioxx** [comment]
Thanks! Can I have a URL to test the same behaviour on my side?
In the meantime I have tried with GitHub (the comment box is recognized as form and the tab never automatically suspended) and the forum of Mozilla Italia with the same results: tabs never automatically suspended.

**Comment by baz357** [comment]
Sure, it's https://forum.next-episode.net/

**Comment by CameosVatican** [comment]
> > Is the option "Never suspend tabs that contain unsaved form inputs" enabled?
> 
> Yes, it is. I just checked again to be sure.

I have this issue too. 🥴

Have lost loads of form-entered data while I popped over to a fresh tab to check something to add to the form.

And I’ve always had TMS set to never suspend a tab when a form is being completed.

I even double-checked after TMS suspended the tab, to see if the setting had somehow been altered; it hadn’t.

Am now terrified of leaving a form partway complete ever again. 🥴

Any ideas?!


[MBP, running BigSur; latest version of Chrome.]

**Comment by baz357** [comment]
> Am now terrified of leaving a form partway complete ever again. 🥴
> 
> Any ideas?!

As a temporary solution, I now always use the "Don't suspend this URL" option when I see the icon in the incorrect state.

**Comment by owenblacker** [comment]
I'm seeing the same thing, also on 7.1.6.2 and Chrome/Windows latest

This also sounds similar to #127

**Comment by baz357** [comment]
Yeah, only hours ago I lost another text because of this bug. :( 

After 3 months with this bug, I would _love_ an update that fixes this.

**Comment by nomennescio** [comment]
Just lost an hour's worth of editing due to this bug, in v7.1.6.2. This is extremely annoying, and I run into this bug regularly, as I have relatively short suspend times.

Besides that; MarvellousSuspender is a wonderful tool, thank you for that!

**Comment by nomennescio** [comment]
Is it possible for MarvellousSuspender to save these fields when suspending? Then even not having set the option would be less disastrous.

**Comment by davidem00** [comment]
I just had this happen on Reddit.  a few paragraphs typed into a comment form (not a new post form).
v7.1.6.2

**Comment by davidem00** [comment]
Still happening on Reddit.  This time it was via a session on Edge (which gets invoked when you open extra tabs from the "MS Store App" version of Reddit).

Lol, ok, nm, I see that it's still on v7.1.6.2
